packages/docs/site/i18n/gu/docusaurus-plugin-content-docs/current/main/about/build.md

```diff
@@ -0,0 +1,83 @@
+---
+title: બિલ્ડ
+slug: /about/build
+description: જાણો કે વર્ડપ્રેસ પ્લેગ્રાઉન્ડ તમને કેવી રીતે પ્રોડક્ટ્સ બનાવવા મદદ કરે છે — લોકલ એન્વાયરમેન્ટ્સ સેટઅપ કરવાથી લઈને થીમ્સ અને નવા ટૂલ્સ બનાવવા સુધી.
+sidebar_class_name: navbar-build-item
+---
+
+# બિલ્ડ
+
+વર્ડપ્રેસ પ્લેગ્રાઉન્ડ તમને ઝડપથી બનાવવા અને શીખવામાં મદદ કરી શકે છે, ભલે મોબાઇલ પર સિગ્નલ ન હોય ત્યારે પણ.
+તમે પ્લેગ્રાઉન્ડનો ઉપયોગ ત્યાં કરી શકો છો જ્યાં તમે સારું કામ કરો છો — પછી તે બ્રાઉઝર હોય, Node.js, મોબાઇલ એપ્સ, VS કોડ અથવા અન્ય ક્યાંય.
+
+## ઝડપથી લોકલ વર્ડપ્રેસ એન્વાયરમેન્ટ સેટ કરવું
+
+તમે તમારા ડેવલપમેન્ટ વર્કફ્લોમાં પ્લેગ્રાઉન્ડને સરળતાથી ઇન્ટિગ્રેટ કરી શકો છો,
+જેથી કોડ ટેસ્ટ કરવા માટે તમે ઝડપથી લોકલ વર્ડપ્રેસ એન્વાયરમેન્ટ લોન્ચ કરી શકો.
+
+તમે આ સીધા જ [ટર્મિનલમાંથી](/developers/local-development/wp-playground-cli) અથવા [તમારા પસંદગીના IDE](/developers/local-development/vscode-extension) માંથી કરી શકો છો.
+
+## બ્લોક થીમ પર કરેલા ફેરફારો સાચવો અને GitHub Pull Requests બનાવો
+
+તમે તમારી પ્લેગ્રાઉન્ડ ઇન્સ્ટન્સને GitHub રેપોઝિટરી સાથે કનેક્ટ કરી શકો છો અને વર્ડપ્રેસ UI મારફતે કરેલા ફેરફારો સાથે Pull Request બનાવી શકો છો,
+[Create Block Theme](https://wordpress.org/plugins/create-block-theme/) પ્લગિનનો ઉપયોગ કરીને.
+
+આ વર્કફ્લો સાથે, તમે તમારા બ્રાઉઝરમાં જ સંપૂર્ણ બ્લોક થીમ બિલ્ડ કરી શકો છો અને તમારા ફેરફારો GitHub પર સાચવી શકો છો,
+અથવા કોઈ હાલની થીમને સુધારી/ફિક્સ કરી શકો છો.
+
+<iframe width="800" src="https://www.youtube.com/embed/94KnoFhQg1g" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<p></p>
+આ વર્કફ્લોના થોડા વધુ ઉદાહરણો:
+
+-   [ડેવલપર અવર્સ: ટેસ્ટિંગ અને ડેમોઝ માટે વર્ડપ્રેસ પ્લેગ્રાઉન્ડ બ્લૂપ્રિન્ટ્સ બનાવવું](https://www.youtube.com/watch?v=gKrij8V3nK0&t=2488s)
+-   [રીકૅપ હોલવે હેંગઆઉટ: પ્લેગ્રાઉન્ડ સાથે થીમ બિલ્ડિંગ, ક્રિએટ-બ્લોક-થીમ પ્લગિન અને GitHub](https://make.wordpress.org/core/2024/06/25/recap-hallway-hangout-theme-building-with-playground-create-block-theme-plugin-and-github/)
+
+## તમારા પ્લેગ્રાઉન્ડ ઇન્સ્ટન્સને લોકલ ફોલ્ડર સાથે સિંક્રોનાઈઝ કરો અને GitHub Pull Requests બનાવો
+
+![સ્ટોરેજ ટાઈપ ડિવાઈસ સ્નેપશોટ](../_assets/storage-type-device.png)
```

Comment:
Please check the latest branch of the documentation, those assets are updated and now are loaded from `packages/docs/site/static/img`. So you don't need to copy the image files anymore.

Reply:
Thanks. I have removed the png file and updated the code to use webp file. Do you want me to add english translations as comments for rest of the `about` pages?